tests/search/reducers/test_search.js

```diff
@@ -0,0 +1,8 @@
+import search from 'search/reducers/search';
+
+describe('search reducer', () => {
+  it('has a null query', () => {
+    const { query } = search(undefined, {type: '@@INIT'});
```

Comment:
No, I just needed it to be any type. I think that's what redux passes to initialize it but I'm not positive on that. I'll change it to something else so that it is obvious it doesn't matter.